composer.json

```diff
@@ -29,7 +29,7 @@
         "psr-7"
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
```

Comment:
So we can only support PHP8.2 if we kick 7.4?

Reply:
I was just wondering, but since PHP7.4 is EOL I will not force this compatibility.